kv/memdb.go

```diff
@@ -125,6 +125,17 @@ var tombstone = []byte{}
 // IsTombstone returns whether the value is a tombstone.
 func IsTombstone(val []byte) bool { return len(val) == 0 }
 
+// MemKeyHandle represents a pointer for key in MemBuffer.
+type MemKeyHandle struct {
+	Passthrough uint16
```

Comment:
How about `Meta`? 
I think is more commonly used and easier to understand.

Reply:
It is an opaque user data instead of metadata of handle. Similar concepts in kqueue and epoll are referred to as udata or data.